comp/netflow/flowaggregator/flowaccumulator.go

```diff
@@ -135,7 +203,9 @@ func (f *flowAccumulator) add(flowToAdd *common.Flow) {
 		f.addRDNSEnrichment(aggHash, flowToAdd.SrcAddr, flowToAdd.DstAddr)
 	} else {
 		// use go routine for hash collision detection to avoid blocking critical path
-		go f.detectHashCollision(aggHash, *aggFlow.flow, *flowToAdd)
+		if !f.skipHashCollisionDetection {
```

Comment:
Profiling shows a significant amount of overhead for the creation of the goroutines to do the hash collision detection, and the collision detection function is pretty short.  As such it probably makes sense to run the hash detection in-line.